pkg/controller/vitessshardreplication/repair_replication.go

```diff
@@ -84,6 +84,19 @@ func (r *ReconcileVitessShard) repairReplication(ctx context.Context, vts *plane
 		return resultBuilder.Result()
 	}
 
+	// If the current master is in a cell that this VitessCluster doesn't
```

Comment:
In theory, couldn't we just place a lock on replication repair if one of the operators gets to it? Or do we not want the cells to interact at all? Otherwise, LGTM.